demos/demo.bat

```diff
@@ -8,6 +8,8 @@
 :: Implemented as a bash script which invokes python so that we can test the
 :: behaviour on exit, which resets default colors again.
 
+cd /d "%~dp0"
+
```

Comment:
Help me understand the motivation for these. They are probably sensible, I just like MPs to be self-explanatory, rather than readers speculating what changes are for.